mhook-lib/mhook.cpp

```diff
@@ -969,14 +969,23 @@ static DWORD DisassembleAndSkip(PVOID pFunction, DWORD dwMinLen, MHOOKS_PATCHDAT
         while ( (dwRet < dwMinLen) && (pins = GetInstruction(&dis, (ULONG_PTR)pLoc, pLoc, dwFlags)) ) 
         {
             ODPRINTF((L"mhooks: DisassembleAndSkip: %p:(0x%2.2x) %s", pLoc, pins->Length, pins->String));
-            if (pins->Type == ITYPE_RET     ) break;
-            if (pins->Type == ITYPE_BRANCH  ) break;
-            if (pins->Type == ITYPE_CALLCC  ) break;
-
+            if (pins->Type == ITYPE_RET     )   break;
+            if (pins->Type == ITYPE_BRANCHCC)   break;
+            if (pins->Type == ITYPE_CALL)       break;

```

Comment:
`ITYPE_CALL` is allowed and is handled later in the code, so this line should be removed. Everything else looks fine.

Reply:
Good point, copied over from the patchfile I use in VCPKG (massive since it also removed trailing whitespace) and apparently copied the wrong line. Sorry about that.